### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (454)
Screenshot (455)
--Name of the website  : Dot Store
-- I made this E-commerce website for book selling.
-- This is the landing page of that website.
-- It has a header for navigation.
-- It has different authors to choose novels of.
-- It has a hero image.
-- It has two cards for new-arrivals.

### CODE REVIEW

File: common-css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    position:relative;
+}
+.secondary-btn{
+    padding: 5px 5px;
+}
+.real-badge{
+    top: 0;
+    right: 0;
+    position: absolute;
+    margin: 14px;
```
Comment (stutisk) [inline common-css/navbar.css]: If value is above then 10 use rem

**Review by stutisk — APPROVED** [review]
Everything else looks good to me

File: common-css/navbar.css (hunk 1 above)
Comment (Jia-A) [reply common-css/navbar.css]: Yes. Changing it.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (dev-enforced) [inline .vscode/settings.json:3]: Can avoid pushing this file to github by having a .gitignore file that helps in avoiding adding things related to github

File: style.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    margin: 15px 0px 0px 0px;
+    background-color: transparent;
+}
+.complete-card {
+    max-width: 40rem;
+    background-color: #E1F0C4;
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.2);
+}
+.hori-img, .main-info-cont{
+    /* border-radius: var(--border-radius); */
```
Comment (dev-enforced) [inline style.css]: Can remove this commented code

File: style.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.responsive-img {
+    height: 450px;
+}
+
+.cards-cont {
+    display: flex;
+    margin: 20px;
+    gap: 100px;
```
Comment (dev-enforced) [inline style.css]: Can use rem here to describe the gap value

File: common-css/navbar.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+.website-name{
+    font-weight: 500;
+    color: var(--primary-color);
+}
+.btn-div{
+    display: inline;
+}
+.nav-btn{
+    display: flex;
+    flex-direction: row;
```
Comment (dev-enforced) [inline common-css/navbar.css:30]: Can remove this property as by default flex-direction is row

File: common-css/navbar.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+.real-badge{
+    top: 0;
+    right: 0;
+    position: absolute;
+    margin: 14px;
+    border-radius: 50%;
+    height: 0.8125rem;
+    width: 0.8125rem;
+    background-color: var(--alert-color);
+    color: whitesmoke;
```
Comment (dev-enforced) [inline common-css/navbar.css]: Can use variables having hex or RGB coded values instead of words for better understanding

File: index.html [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+                <button class="small-text btn secondary-btn-border">Nicholas Sparks</button>
+            </div>
+            <div class="img-cont">
+                <img src="./assets/colleen-hoover.png" class="round-img">
+                <button class="small-text btn secondary-btn-border">Colleen Hover</button>
+            </div>
+        </div>
+    </div>
+    <div class="hero">
+        <img src="./assets/hero-img.jpg" class="responsive-img">
```
Comment (dev-enforced) [inline index.html]: Do add an alt attribute describing the content of the image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+                <button class="small-text btn secondary-btn-border">Khaled Hosseini</button>
+            </div>
+            <div class="img-cont">
+                <img src="./assets/nicholas.jpg" class="round-img">
+                <button class="small-text btn secondary-btn-border">Nicholas Sparks</button>
+            </div>
+            <div class="img-cont">
+                <img src="./assets/colleen-hoover.png" class="round-img">
+                <button class="small-text btn secondary-btn-border">Colleen Hover</button>
+            </div>
```
Comment (dev-enforced) [inline index.html]: Same as below alt attribute

**Review by dev-enforced — APPROVED** [review]
Deploy the website on Netlify or Vercel so that the reviewers can get a better idea of the work you have done while developing the project. The code looks good. Provided some minor suggestions using which you can develop the product

**Pull request merged**